src/main/java/teachingtutorials/guis/locationcreatemenus/StepEditorMenu.java

```diff
@@ -42,10 +44,15 @@ public StepEditorMenu(TeachingTutorials plugin, User user, Step step, LocationSt
         this.step = step;
         this.locationStep = locationStep;
 
-        this.book = new ItemStack(Material.WRITABLE_BOOK, 1);
-        BookMeta bookMeta = (BookMeta) book.getItemMeta();
-        bookMeta.setTitle(step.getName());
-        this.book.setItemMeta(bookMeta);
+        this.instructionsBook = new ItemStack(Material.WRITABLE_BOOK, 1);
+        BookMeta instructionsBookMeta = (BookMeta) instructionsBook.getItemMeta();
+        instructionsBookMeta.setTitle(step.getName());
+        this.instructionsBook.setItemMeta(instructionsBookMeta);
+
+        this.videoLinkBook = new ItemStack(Material.WRITABLE_BOOK, 1);
+        BookMeta videoLinkBookMeta = (BookMeta) videoLinkBook.getItemMeta();
+        videoLinkBookMeta.setTitle(step.getName());
+        this.videoLinkBook.setItemMeta(videoLinkBookMeta);
```

Comment:
I feel like that could be refactored, you've basically used the exact same code for both books.

Reply:
Moved this into the initialisation of the TextEditorBookListeners itself, and now both book types share the same code.